src/services/auth.js

```diff
@@ -1,21 +1,25 @@
 import Axios from 'axios'
 import { VueAuthenticate } from 'vue-authenticate'
 
-const vueAuthInstance = new VueAuthenticate(Axios, {
-  providers: {
-    oauth2: {
-      name: 'ownCloud',
-      clientId: 'Cxfj9F9ZZWQbQZps1E1M0BszMz6OOFq3lxjSuc8Uh4HLEYb9KIfyRMmgY5ibXXrU',
-      url: 'http://deepdiver:8080/index.php/apps/oauth2/api/v1/token',
-      authorizationEndpoint: 'http://deepdiver:8080/index.php/apps/oauth2/authorize',
-      responseType: 'token',
-      responseParams: {
-        code: 'code',
-        clientId: 'client_id',
-        redirectUri: 'redirect_uri'
+export function initVueAuthenticate (config) {
+  if (config) {
+    return new VueAuthenticate(Axios, {
+      providers: {
+        oauth2: {
+          name: 'ownCloud',
+          clientId: config.clientId || '',
```

Comment:
I think there are no reasonable default - if not set -> :boom:

Reply:
https://github.com/owncloud/phoenix/pull/303#discussion_r230759679